### PR TITLE
add collabora extra param

### PR DESCRIPTION
**Pull request by ScharfViktor**
this fix can help to solve this issue https://github.com/owncloud/ocis/issues/9706#issuecomment-2257756493 with local running collabora v.24

related https://github.com/owncloud/web/pull/11301

### CODE REVIEW

**Comment by micbar** [comment]
We need to create an env var for this in the .env file because we should not set the default to insecure.

**Review by micbar — CHANGES_REQUESTED** [review]
Please make the default secure.

**Review by AlexAndBear — APPROVED** [review]

File: deployments/examples/ocis_full/collabora.yml [hunk 1]
```diff
@@ -55,7 +55,7 @@ services:
     environment:
       aliasgroup1: https://${WOPISERVER_DOMAIN:-wopiserver.owncloud.test}:443
       DONT_GEN_SSL_CERT: "YES"
-      extra_params: --o:ssl.enable=false --o:ssl.termination=true --o:welcome.enable=false --o:net.frame_ancestors=${OCIS_DOMAIN:-ocis.owncloud.test}
+      extra_params: --o:ssl.enable=false --o:ssl.ssl_verification=${SSL_VERIFICATION:-true} --o:ssl.termination=true --o:welcome.enable=false --o:net.frame_ancestors=${OCIS_DOMAIN:-ocis.owncloud.test}
```
Comment (AlexAndBear) [inline deployments/examples/ocis_full/collabora.yml:58]: > --o:ssl.enable=false

This doesn't need to be enabled to make it "secure" as Michael asked for it

Comment (ScharfViktor) [reply deployments/examples/ocis_full/collabora.yml]: thanks 👍  I did it by analogy with `INSECURE=true` 
by default `SSL_VERIFICATION=true` `SSL_ENABLE=true` 
for development/testing is set to false

File: deployments/examples/ocis_full/.env [hunk 2]
```diff
@@ -129,6 +129,9 @@ COLLABORA_ADMIN_USER=
 # Admin password for Collabora.
 # Defaults to "admin".
 COLLABORA_ADMIN_PASSWORD=
+# If you're on an internet-facing server, enable SSL verification for Collabora Online.
+# Please comment out the following line:
+SSL_VERIFICATION=false
```
Comment (micbar) [inline deployments/examples/ocis_full/.env:136]: I would prefer a COLLABORA_ prefix

Comment (ScharfViktor) [reply deployments/examples/ocis_full/.env]: done

**Review by micbar — APPROVED** [review]

**Pull request merged**